src/components/Footer.js

```diff
@@ -71,13 +70,13 @@ export default function Footer() {
               <h2 className="mb-6 text-xl font-serif uppercase">Support</h2>
               <div className="font-medium md:text-lg flex flex-row ">
                 <div className="flex flex-col mr-8">
-                  <Link
+                  <a
```

Comment:
remind me why do we change the link to a tag?

Reply:
I was getting a warning about having Link tags pointing to a link outside of the gatsby project. Links are strictly for local routing